src/parade/LetterLink.js

```diff
@@ -7,7 +7,7 @@ const LetterLink = ({ letter, isDisabled, isActive }) => {
     ? { isDisabled }
     : { href: `#parade-group-${letter}`, isDisabled }
   return (
-    <StyledLetterLink isActive={isActive} {...letterAttributes}>
+    <StyledLetterLink isActive={isActive} {...letterAttributes} role="link">
```

Comment:
Try without this explicit role 😄 this role should be applied by default to an `a` element